Nimble/CallRecorder.swift

```diff
@@ -0,0 +1,312 @@
+
+/* bug in Swift causes every enum WITHOUT an associated value's "description" to be the first declared
+enum value WITHOUT an associated value's description
+i.e. -> since ".DontCare" is the first enum value then the default description for ".DontCare",
+".NonNil", and ".Nil" will all be "DontCare" -> must override to fix issue by conforming to "CustomStringConvertible" 
+    This is not reproducable in a Playground */
+
+public enum Argument : CustomStringConvertible {
+    case DontCare
+    case NonNil
+    case Nil
+    case InstanceOf(type: Any.Type)
+    case InstanceOfWith(type: Any.Type, option: ArgumentOption)
+    case KindOf(type: AnyObject.Type)
+    
+    public var description: String {
+        switch self {
+        case .DontCare:
```

Comment:
I was going to use 'Anything' but I didn't want it to conflict with idea of swift's 'Any' or 'AnyObject'. Also I thought it would read better when using 'toNot'. If you / everyone still agrees that 'Anything' would be better then I'll change it.